README.md

```diff
@@ -61,6 +61,8 @@ go run -mod=vendor main.go daemon \
 
 - `--service.s3.bucket`: (Required) AWS S3 Bucket name.
 - `--service.s3.region`: (Required) AWS S3 Region name.
+- `--service.s3.endpoint`: (Optional) Custom S3 endpoint URL.
+- `--service.s3.force-path-style`: (Optional, defaults to `false`) Enable path-style S3 URLs.
```

Comment:
Would be nice to mention here what the default behaviour with `false` is. From reading this up, I guess it's hosted-style URLs?

Reply:
It's a setting that you only enable when you know what you are doing. The default, nowadays, is to have URLs as follows:

`https://bucket-name.s3.region-code.amazonaws.com`

However, there are still buckets on AWS and other providers that use the path style pattern:

`https://s3.region-code.amazonaws.com/bucket-name/key-name`

I.e. the the default is 'false' anyways. Previously it was just not explicit. Would you still like to add an explaination? If so, do you have a suggestion?